Mathlib/Analysis/Calculus/ContDiff.lean

```diff
@@ -550,7 +550,7 @@ We show that the composition of `C^n` functions is `C^n`. One way to prove it wo
 the `n`-th derivative of the composition (this is Faà di Bruno's formula) and check its continuity,
 but this is very painful. Instead, we go for a simple inductive proof. Assume it is done for `n`.
 Then, to check it for `n+1`, one needs to check that the derivative of `g ∘ f` is `C^n`, i.e.,
-that `Dg(f x) ⬝ Df(x)` is `C^n`. The term `Dg (f x)` is the composition of two `C^n` functions, so
+that `Dg(f x) * Df(x)` is `C^n`. The term `Dg (f x)` is the composition of two `C^n` functions, so
```

Comment:
Can you double-check the comments in this file? I believe they might have used the character `⬝` by accident here.